### ISSUE TITLE
Create sample data for DBs to load in test setups

### ISSUE THREAD

**Issue by oleggtro** [MathisBurger/CodeCanvas#71]
It would probably make sense to generate some sample data so that new instances used for development are prefilled with user accounts, groups, assignments, etc.

I believe that would help dev cycles a lot as we don't need to create all those objects by hand

**Comment by MathisBurger** [comment]
Yeah, this would make sense. But due to the lack of development on this project this would not make that much sense. Furthermore setting up some data manually isn`t that hard too
